python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -744,13 +746,6 @@ def main():
             sys.stderr.write('"setproctitle" missing.\n')
             exit_code = 7
 
-        try:
-            from Crypto.PublicKey import _fastmath
-            exit_code = egg_check(_fastmath) | exit_code
-        except:
-            sys.stderr.write('Crypto.PublicKey._fastmath missing. No GMP?\n')
```

Comment:
there is no  fastmath support in cryptodome?

Reply:
The tests for PyCryptodome are in `chevah_build`. There is support for using GMP for faster math.